codestyle.xml

```diff
@@ -0,0 +1,23 @@
+<code_scheme name="spinnaker">
```

Comment:
If this formatting template is applied to the currently committed codebase, are changes made?